DNN Platform/Modules/DDRMenu/SkinObject.cs

```diff
@@ -18,8 +17,16 @@ namespace DotNetNuke.Web.DDRMenu
 
     public class SkinObject : SkinObjectBase
     {
+        private readonly ILocaliser localiser;
         private MenuBase menu;
 
+        /// <summary>Initializes a new instance of the <see cref="SkinObject"/> class.</summary>
+        /// <param name="localiser">The tab localizer.</param>
+        public SkinObject(ILocaliser localiser)

```

Comment:
Will this just work because it's a webforms control or do we need to add an obsolete parameterless constructor here just in case ?